org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XSDDocumentationSource.java

```diff
@@ -0,0 +1,23 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.settings;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * XSD documentation source enum
+ */
+public enum XSDDocumentationSource {
+	@SerializedName("Both") both,
```

Comment:
Why are you using SerializedName?

Reply:
I guess I don't necessarily need it for `both`, but I use it for `documentation` and `appinfo`:
https://github.com/xorye/lsp4xml/blob/914dc9e2c530e7d9c269dfebc3934de849ad0719/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XSDDocumentationSource.java#L21-L22

because the enum value in vscode-xml is like so:
https://github.com/redhat-developer/vscode-xml/blob/e5be58f2c56dbb7132297086534811ce0cbff3f9/package.json#L238-L239

Maybe instead, we should remove `@SerializedName` and change the enum names to `schemaDocumentation` and `schemaAppinfo`?

Reply:
I find XSDDocumentationSource is not very understandable and I think enum should short and without spaces. Here a suggestion:

```json
"xml.preferences.showShemaDocumentationType": {
          "type": "string",
          "default": "documentation",
          "enum": [
            "documentation",
            "appinfo",
            "both"
          ]
...

@fbricon what do you think about that?

Reply:
documentation, appinfo, all

Reply:
I agree that all is better, but what do you think about 

 * renaming `XSDDocumentationSource `to `ShemaDocumentationType`
 * renaming `xml.preferences.documentationSource` to `xml.preferences.showShemaDocumentationType`

Reply:
ok.

also need a `none` choice

Reply:
fixed